### DISCUSSION TITLE
Set WalletConnect theme?

### DISCUSSION THREAD

**Discussion by KevinLu** [wevm/wagmi#1846] [Q&A]
Since WalletConnect v2 uses a new modal component, how can I set the theme with wagmi like shown here: https://docs.walletconnect.com/2.0/web3modal/standalone/actions#web3modalsettheme

**Comment by SaadNiaziDev** [comment]
How u are creating web3modal instance? Are u using standalone or react web3modal?

**Reply by KevinLu** [reply]
I'm using the Walletconnect connector exported from wagmi like this:
```js
const client = createClient({
  connectors: [
    new WalletConnectConnector({
      chains,
      options: {
        qrcode: true,
        version: '2',
        projectId: 'my id',
      },
    }),
  ],
  provider,
  webSocketProvider,
});
```

I'm not using Web3Modal standalone or React version at all, I let wagmi handle that.

**Reply by KevinLu** [reply]
The behavior right now is the modal takes the user's system theme, but my dApp has the ability to set theme (light/dark mode) irrespective of the system theme. So I would like to control the modal theme programatically.

**Comment by SaadNiaziDev** [comment]
if using @web3modal/html then follow this..
https://docs.walletconnect.com/2.0/web3modal/html-js/actions#web3modalsettheme

if using @web3modal/react then follow this..
https://docs.walletconnect.com/2.0/web3modal/react/hooks#useweb3modaltheme

if using @web3modal/standalone then follow this..
https://docs.walletconnect.com/2.0/web3modal/standalone/actions#web3modalsettheme

Pretty straight forward explaination. just import or use instance of web3modal
